### PR TITLE
Add delete account option in accounts tab in settings

### LINKED ISSUE #1404: Add delete account option in account's tab as on account.susi
**Actual Behaviour**
No option to delete account in account's tab.

**Expected Behaviour**
Delete account option should be there in account tab like https://github.com/fossasia/accounts.susi.ai/pull/297

**Screenshots of the issue**
d1

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by PrP-11**
Fixes #1404 

Changes: Add delete account option in account tab

Demo Link: https://pr-1444-fossasia-susi-web-chat.surge.sh

Screenshots for the change: 
d1

### CODE REVIEW

File: src/components/ChatApp/Settings/Settings.react.js [hunk 1]
```diff
@@ -2143,6 +2143,35 @@ class Settings extends Component {
...
+                ''
+              ) : (
+                <div style={{ marginRight: '20px' }}>
+                  {UserPreferencesStore.getTheme() === 'light' ? (
+                    <hr
+                      className="break-line-light"
+                      style={{ height: '2px', marginTop: '25px' }}
+                    />
+                  ) : (
+                    <hr className="break-line-dark" />
```
Comment (fragm3) [inline src/components/ChatApp/Settings/Settings.react.js]: No style for dark theme? Correct it!

File: src/components/ChatApp/Settings/Settings.react.js [hunk 2]
```diff
@@ -2143,6 +2143,35 @@ class Settings extends Component {
...
+                  )}
+
+                  <p
+                    style={{
+                      textAlign: 'center',
+                      marginTop: '20px',
+                      display: 'flex',
+                      justifyContent: 'center',
```
Comment (fragm3) [inline src/components/ChatApp/Settings/Settings.react.js]: No need to centre it, align it like rest of the fields

Comment (PrP-11) [reply src/components/ChatApp/Settings/Settings.react.js]: @fragm3 The style is followed the same way as on accounts page. We should follow similar alignment

File: src/components/ChatApp/Settings/Settings.react.js (hunk 1 above)
Comment (PrP-11) [reply src/components/ChatApp/Settings/Settings.react.js]: @fragm3 Corrected it.

**Pull request merged**